pom.xml

```diff
@@ -1964,12 +1976,6 @@
                 <scope>compile</scope>
             </dependency>
 
-            <dependency>
```

Comment:
I notice you have removed this dependency again -- it was the reason for the last PR build failure and this one -- we need it. It defines a transient dependency

Reply:
Hi Nigel, from the build failure it shows the dependency is missing from artifactory of odpi.jfrog as we can build it locally. Would that make sense for you to have a look on it? Thanks 
`org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project graph-repository-connector: Could not resolve dependencies for project org.odpi.egeria:graph-repository-connector:jar:1.4-SNAPSHOT: Could not find artifact com.sleepycat:je:jar:7.5.11 in central (https://odpi.jfrog.io/odpi/egeria-release)`

Reply:
If we have the dependency as it was we can open the graph in gremlin console it needs an older version of this dependency. Since it has it embedded in some library there is no need of having it explicitly. Locally is building with no issues.

Reply:
@planetf1  Berkeley is using this one:
    - com.sleepycat:je:jar:7.5.11:compile

If we can find a version that is compatible with this then we can added as a separate dependency with no issue.

Reply:
> Hi Nigel, from the build failure it shows the dependency is missing from artifactory of odpi.jfrog as we can build it locally. Would that make sense for you to have a look on it? Thanks
> `org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project graph-repository-connector: Could not resolve dependencies for project org.odpi.egeria:graph-repository-connector:jar:1.4-SNAPSHOT: Could not find artifact com.sleepycat:je:jar:7.5.11 in central (https://odpi.jfrog.io/odpi/egeria-release)`

It's missing in maven central too - which would cause an issue for anyone else using berkeley. We need to use a version oracle has pushed to the repos. See https://github.com/odpi/egeria/issues/1116

Adding to our artifactory at odpi isn't sufficient

Reply:
> If we have the dependency as it was we can not open the graph in gremlin console it needs an older version of this dependency. Since it has it embedded in some library there is no need of having it explicitly. Locally is building with no issues.
 
Are you saying the newer berkely is causing an issue with gremlin? I know @grahamwallis uses gremlin, and that dependency has been in place for quite a while

Can you elaborate on this?
Do you have any instructions on how to repro? I'm happy to offer an additional opinion, but don't know how to setup the environment (simply)? Or can you share a transcript of what you see when it 'fails'? And how does your local build's dependency tree differ to master (or this proposal)?

Reply:
To reproduce if you want you can keep the sleepycat dependency in with the latest version as it was and start the open lineage services. I can send you the post call if you like. Then if you try to open the graph from the gremlin console you will see this.

Caused by: com.sleepycat.je.VersionMismatchException: (JE 7.5.11) /Users/wd38tc/Workspace/egeria/egeria-lineage-repositories/bufferGraph/berkeley Expected log version 15 or earlier but found 17 VERSION_MISMATCH: The existing log was written with a version of JE that is later than the running version of JE, the log cannot be read. Environment is invalid and must be closed. Couldn't open file /Users/Workspace/egeria/egeria-lineage-repositories/bufferGraph/berkeley/00000000.jdb

Reply:
Thanks. Yes REST would be good.

Are you starting from a clean environment? ie if you have no lineage graph at all - completely clean config of egeria. It looks as if there is an incompatibility between files (the tx log) written with one version, and read by another?

Another possibility is that you have inadvertantly included 2 different versions of berkeley - but you would see that in the mvn dependency:tree plus our build validation *should* prevent it.

Or are you running both old and new code?